src/app/public/modules/repeater/repeater-item.component.ts

```diff
@@ -106,18 +128,36 @@ export class SkyRepeaterItemComponent implements OnDestroy, OnInit {
 
   public slideDirection: string;
 
+  public set tabIndex(value: number) {
+    this._tabIndex = value;
+  }
+
+  public get tabIndex(): number {
+    return this._tabIndex;
+  }
```

Comment:
We can probably get away with a simple property.
```
public tabIndex: number = -1;
```

Reply:
Done.